pilot/lib/pilot/tester_manager.rb

```diff
@@ -43,6 +40,26 @@ def add_tester(options)
       end
     end
 
+    def add_tester_to_groups(tester: nil, app: nil)
+      default_external_group = app.default_external_group
+      if default_external_group.nil? && config[:groups].nil?
+        UI.user_error!("The app #{app.name} does not have a default external group. Please make sure to pass group names to the `:groups` option.")
+        return
+      end
+
+      default_external_group.add_tester!(tester) unless default_external_group.nil?
```

Comment:
@ohayon  does this only happen if user has 0 groups? as i get it, it seems the first created is also the default.
can't we create a "External" group if no default is found - and do a `UI.success`reporting it?